src/Models/HydrostaticFreeSurfaceModels/fft_based_implicit_free_surface_solver.jl

```diff
@@ -28,7 +28,7 @@ representing an implicit time discretization of the linear free surface evolutio
 for a fluid with constant depth `H`, horizontal areas `Az`, barotropic volume flux `Q★`, time
 step `Δt`, gravitational acceleration `g`, and free surface at time-step `n`, `ηⁿ`.
 """
-function FFTImplicitFreeSurfaceSolver(arch, grid, settings)
+function FFTImplicitFreeSurfaceSolver(arch, grid, gravity::Number, settings)
```

Comment:
```suggestion
function FFTImplicitFreeSurfaceSolver(arch, grid, gravitational_acceleration::Number, settings)
```

?? what is `gravity` ?